src/index.js

```diff
@@ -0,0 +1,56 @@
+import './style.css';
+
+let gameId = '';
+const addNew = document.querySelector('.add-new');
+const loadGame = document.querySelector('.refresh-btn');
+const createGame = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/',
+    {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: 'fifa-2023',
+      }),
+    });
+  const data = await response.json();
+  [, , , gameId] = data.result.split(' ');
+  return gameId;
+};
+
+const addScore = async (e) => {
+  e.preventDefault();
+  const user = document.querySelector('.name').value;
+  const score = document.querySelector('.score').value;
+  const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+    {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: JSON.stringify({
+        user,
+        score,
+      }),
+    });
+  const data = await response.json();
+  console.log(data);
+};
+
+const loadNewGame = async (e) => {
```

Comment:
- [ ] Aside from this, when the page is refreshed, the array returning from the API comes back empty as a result there are no scores to show on the page even when the refresh button is clicked. However, since data was already posted to the API, they should still be retrieved even when the page is refreshed. So kindly fix this as well :+1: 

[Screencast from 06-21-2023 10:14:14 AM.webm](https://github.com/olanikegloria/leaderboard-project/assets/102699958/4ee03424-0e5a-4009-b02c-2a180cceda6d)